package EquipSuitConfig;

message Material
{
	required uint32 eqid	= 1;  //材料id
	required uint32 count	= 2;   //数量
}

message SuitEquip
{
	required uint32 preeqid		= 1;   //前置装备id
	required uint32 suiteqid	= 2;   //套装id
	repeated Material material	= 3;   //材料
}

message Property
{
	required uint32 num		= 1;  //套装个数
	repeated uint32 ptype	= 2; //属性类型
	repeated uint32 value	= 3;  //属性值
}

message SuitProperty
{
	required uint32 type		= 1; //套装类型
	repeated Property property	= 2;  //属性
}

message EquipSuit
{
	repeated SuitEquip suitequip		= 1;  //套装条件
	repeated SuitProperty suitproperty	= 2;  //套装属性
}